src/packet_router/handshake_init.rs

```diff
@@ -0,0 +1,11 @@
+use super::packet::Packet;
+
+// Called upon handshake between two servers
+pub fn init_handshake(p: Packet, state: &mut u64){
+    println!("Handshake packet: {:?}", p);
+    
+    *state = match p.clone() {
```

Comment:
This is fine for now but I think having the state be passed around as mutable right now isn't great- IMO the state that is being passed around isn't mutable, state doesn't change mid-request, it only changes at the end of a request in a few special cases. Because of that it would be better if we just made a call here to change the state for the next request. I'll throw up a ticket for it